nemo/collections/cv/modules/global_avg_pool2d.py

```diff
@@ -0,0 +1,12 @@
+from nemo.core.classes.module import NeuralModule
+
+
+class GlobalAvgPool2d(NeuralModule):
+    def __init__(self, dim=(2, 3), keepdim=False):
```

Comment:
First assert inside forward that x.ndim == 4 before calling this.

Reply:
done